### PR TITLE
left join

### PR DESCRIPTION
**Pull request by sarastrasner**
- [X] Top-level README “Table of Contents” is updated
 - [X] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [X] README for this challenge is complete
     - [X] Summary, Description, Approach & Efficiency, Solution
     - [X] Link to code
     - [X] Picture of whiteboard

### CODE REVIEW

File: javascript/leftJoin/left.join.test.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const leftJoin = require('./left-join');
+
+describe('Left Join', () => {
+  it('join two tables', () => {
+    let table1 = [{'fond':'enamored'}, {'wrath':'anger'},{'diligent':'employed'},{'outfit':'garb'},{'guide':'usher'}];
+    let table2 =[{'fond':'averse'}, {'wrath':'delight'},{'diligent':'idle'},{'guide':'follow'},{'flow':'jam'}];
+    let result= leftJoin(table1,table2);
+    expect(result).toBeDefined();
+  });
+});
```
Comment (adrianhuebner) [inline javascript/leftJoin/left.join.test.js:10]: Once you have a chance to come back to this and get the test passing I will regrade for you!
